packages/cli/src/controllers/project.controller.ts

```diff
@@ -83,7 +84,8 @@ export class ProjectController {
 
 	@Get('/my-projects')
 	async getMyProjects(
-		req: ProjectRequest.GetMyProjects,
+		req: AuthenticatedRequest,
+		_res: Response,
```

Comment:
Do we need this if it's not used?

```suggestion
```

Reply:
once we have migrated all endpoints to use DTOs, we can update the controller registry to stop [passing in the first two params](https://github.com/n8n-io/n8n/blob/master/packages/cli/src/decorators/controller.registry.ts#L89). But until then, we need to keep these to avoid breaking controllers that have not been ported over yet.